ccm_web/client/src/components/Home.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Grid from '@material-ui/core/Grid'
+
+import FeatureCard, { FeatureCardProps } from './FeatureCard'
+
+import AccountCircleOutlinedIcon from '@material-ui/icons/AccountCircleOutlined'
+import LibraryBooksOutlinedIcon from '@material-ui/icons/LibraryBooksOutlined'
+import MergeTypeIcon from '@material-ui/icons/MergeType'
+import PersonAddIcon from '@material-ui/icons/PersonAdd'
+import PersonAddOutlinedIcon from '@material-ui/icons/PersonAddOutlined'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+    minWidth: 275,
+    padding: 25
+  }
+}))
+
+function Home (): JSX.Element {
+  const classes = useStyles()
+
+  const mergeSectionProps: FeatureCardProps = {
+    title: 'Merge Sections',
+    description: 'Combine sections into one Canvas site for easier management',
+    icon: <MergeTypeIcon fontSize='large'/>,
+    ordinalty: 1
+  }
+
+  const gradebookToolsProps: FeatureCardProps = {
+    title: 'Gradebook Tools',
+    description: 'Trim the gradebook from Canvas, or trim the gradebook from a third party to correct format',
+    icon: <LibraryBooksOutlinedIcon fontSize='large'/>,
+    ordinalty: 2
+  }
+
+  const addSectionsGroupsProps: FeatureCardProps = {
+    title: 'Add Groups',
+    description: 'Add groups through csv files into your own course',
+    icon: <AccountCircleOutlinedIcon fontSize='large'/>,
+    ordinalty: 3
+  }
+
+  const addUMUsersProps: FeatureCardProps = {
+    title: 'Add UM Users',
+    description: 'Add UM users to your available sections',
+    icon: <PersonAddIcon fontSize='small'/>,
+    ordinalty: 4
+  }
+
+  const addNonUMUsersProps: FeatureCardProps = {
+    title: 'Add Non-UM Users',
+    description: 'Enroll non-UM users to your available sections',
+    icon: <PersonAddOutlinedIcon fontSize='small'/>,
+    ordinalty: 5
+  }
+
+  const cards: FeatureCardProps[] = [mergeSectionProps, gradebookToolsProps, addSectionsGroupsProps, addUMUsersProps, addNonUMUsersProps]
```

Comment:
Not a big difference, but you could just define these all at once, rather than using variables. Also, `ordinality` idea is nice, but we may as well just give them IDs (e.g. `id: 0`).

Reply:
I don't like the idea of ordinality being tied to something not intended for that purpose, but they probably should have an id b/c i was using the title as the key.

Reply:
I see your point. Thanks for adding the `id`.